validator-guide-new.md

```diff
@@ -200,6 +200,29 @@ if you get an error that host cannot be reached over SSH, please wait a minute a
 
 3. open the url for `NETSTAT_SERVER` and check if your node appeared in the list
 
+### Obtaining enode uri for Master of Ceremony
+Login to the node and get enode from parity logs:
+```
+ssh root@192.0.2.1
+grep enode /home/bootnode/logs/parity.log
```

Comment:
home folder name are role-dependent, so this should be `/home/validator/...`

Reply:
yes, sorry I forgot to change that out.